Application/EdFi.Ods.AdminApi/Infrastructure/Helpers/ConnectionStringHelper.cs

```diff
@@ -17,7 +17,7 @@ public static bool ValidateConnectionString(string databaseEngine, string? conne
         {
             try
             {
-                SqlConnectionStringBuilder sqlConnectionStringBuilder = new SqlConnectionStringBuilder(connectionString);
+                _ = new SqlConnectionStringBuilder(connectionString);
             }
             catch (ArgumentException)

```

Comment:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqlconnectionstringbuilder.-ctor?view=sqlclient-dotnet-standard-5.2#microsoft-data-sqlclient-sqlconnectionstringbuilder-ctor(system-string)

Looks like this constructor (line 20) can throw three different exception types. This might be a good opportunity to handle all three, instead of just `ArgumentException`.